pkg/connector/manifest_test.go

```diff
@@ -11,8 +11,13 @@ var _ = Describe("Manifest template", func() {
 			template, err := GetManifestTemplate()
 			Expect(err).ToNot(HaveOccurred())
 
```

Comment:
we should have unhappy path tests here too, right now we seem to only be testing one path.

tests are also missing for `WriteResources`

Reply:
Adding an unhappy path would be difficult here as this test is actually ensuring the URLs are up. 

>tests are also missing for WriteResources

I missed adding them in the initial PR and had planned on adding them later. Thanks for the reminder, I have added them now.